app/adminPage.html

```diff
@@ -91,6 +93,17 @@ <h4 class="search-heading">Filter Results</h4>
                     <tbody class="score_list user-table"></tbody>
                 </table>
             </div>
+            <form class="new-test-container" id="testForm" method="post">
+                <h2>Add new test</h2>
+                <input name="testName" type="text" id="testName" placeholder="Test name">
+                <input type="submit" class="btn primary-button new-test-button submit" value="Add new test">
+                <div id="message-target"></div>
+                <div class="buttons">
+                    <div class="messageArea">
+                        <span id="message"></span>
```

Comment:
this is quite a generic id, on a page full of different inputs etc id use something more specific

Reply:
changed to testSubmitConfirmation